frontend/package.json

```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "@material-ui/core": "^3.9.4",
     "@material-ui/icons": "^3.0.2",
+    "@material-ui/styles": "^4.11.2",
```

Comment:
Can you remove this?

Reply:
The contained module jss is required, so I could remove this easily. And I think adding this module will not cause harm?